src/plugins/plot/configuration/XAxisModel.js

```diff
@@ -126,7 +126,7 @@ export default class XAxisModel extends Model {
 
 /**
 @typedef {import("./Model").ModelType<{
-    range: NumberRange
+    range?: NumberRange
```

Comment:
What is this change for?

Reply:
This just better depicts the type definition of the model object. Because if you look at the `defaultModel` method, it does not return a value for all properties, so some of them can be `undefined` at some points. So the `?` symbol there makes it undefinable. F.e. `foo: number` means foo can only be `number`, but `foo?: number` means foo is `number | undefined` (number or undefined essentially).